### DISCUSSION TITLE
Import doesn't seem to work

### DISCUSSION THREAD

**Discussion by Enigma014** [mrdbourke/pytorch-deep-learning#168] [Q&A] [answered]
I'm writing the following code in Google Colab :
 1 import torch
 2 import pandas as pd
 3 import numpy as numpy
 4 import matplotlib.pyplot as plt
 5 print(torch.__version__) 
and this is the error I get.
line 1
    1 import torch
           ^
SyntaxError: invalid syntax 
Help!

**Comment by ghost** [comment, marked as answer]
Write `torch.__version__` instead of `torch.version`

![image](https://user-images.githubusercontent.com/87841705/200858180-887047b4-1db4-4e73-8656-bbdf665e8005.png)
